lib/active_shipping/carrier.rb

```diff
@@ -92,6 +92,16 @@ def find_tracking_info(tracking_number, options = {})
       raise NotImplementedError, "#find_tracking_info is not supported by #{self.class.name}."
     end
 
+    # Get a list of services available for the a specific route
+    #
+    # @param origin_country [String] The country of origin
+    # @param destination_country [String] The destination country
+    # @return [Array<String>] A list of names of the available services
+    #
+    def available_services(origin_country, destination_country)
```

Comment:
Discussion: Add `options = {}` to be consistent and for possible future options such as optionally providing?